CONTRIBUTING.md

```diff
@@ -0,0 +1,271 @@
+# Contributing
+
+Thank you for your interest in contributing to the project! This project is the result of the passion and hard work of
+many contributors, and we appreciate your help! To get started, please read this document to familiarize yourself with
+the project.
+
+-   [Contributing](#contributing)
```

Comment:
github autogenerates a ToC so you don't need this
![image](https://github.com/vscode-neovim/vscode-neovim/assets/1359421/2e5c6c0d-6aa7-4d0c-8e41-0d62e2b548ef)